android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -208,6 +226,9 @@ public Completable resendEmailVerification(@NonNull final String email) {
                 }).doOnError(throwable -> {
                     // a 412 is a successful signin
                     if (throwable instanceof ConsentRequiredException) {
+                        forConsentedUsersApiAtomicReference.set(
+                                apiClientProvider.getClient(ForConsentedUsersApi.class, signIn)
+                        );
```

Comment:
That's fine, but we need at least one test case that covers this. Whether we update the existing test or create a new test, that's immaterial.